modules/ROOT/pages/comments-commands-events-apis.adoc

```diff
@@ -10,3 +10,34 @@
 The Comments plugin provides the following {productname} commands.
 
 include::partial$commands/comments-cmds.adoc[]
+
+== APIs
+
+The {pluginname} plugin provides the following API.
+
+[[getEventLog]]
+=== getEventLog()
+
+Returns a log of all `CommentChange` events. These events are dispatched whenever:
```

Comment:
This isn't correct... the `CommentChange` is an event fired when something changes, it's not directly related to what's returned by this API. The `CommandChange` event should instead be in an `Events` heading on this page above `Commands` like this: https://www.tiny.cloud/docs/tinymce/6/introduction-to-tiny-spellchecker/#events. It also needs adding to this page: https://www.tiny.cloud/docs/tinymce/6/events/#plugin-events